Documentation/kubernetes/install.rst

```diff
@@ -16,6 +16,80 @@ the :ref:`k8scompatibility` section for kubernetes API version compatibility.
 
 .. _admin_mount_bpffs:
 
+Running Kubernetes with CRD Validation (Recommended)
+====================================================
+
+Custom Resource Validation was introduced in Kubernetes since version ``1.8.0``.
+This is still considered an alpha feature in Kubernetes ``1.8.0`` but since
+``v1.0.0-rc3``, Cilium creates, or updates in case it exists, the Cilium Network
+Policy Resource Definition with the validation schema embedded. This allows the
+validation of CNP to be made on the kube-apiserver for the most common user
+errors.
+
+To enable this feature the flag ``--feature-gates=CustomResourceValidation=true``
+must be set when starting kube-apiserver. As for Cilium, it's already included
+by default since ``v1.0.0-rc3`` so no further actions are necessary.
+
+By running ``kubectl get crd ciliumnetworkpolicies.cilium.io -o json`` it is
+possible to find out if CNP resource definition contains the validation schema.
```

Comment:
Perhaps restructure this sentence to be "to do this, run that"
```
To verify that the CNP resource definition contains the validation schema, run the following command:
``kubectl get crd ciliumnetworkpolicies.cilium.io -o json`
```

Reply:
Done